CHANGELOG.rst

```diff
@@ -2,6 +2,13 @@ Changelog
 =========
 
 
+.. _v2-1-1:
+
+2.1.1 - 2017-10-11
+~~~~~~~~~~~~~~~~~~
+
+* Fixed support for installing with ``pip`` 8.1.1.
```

Comment:
What version of pip/setuptools added `platform_python_implementation`? Are we actually compatible with pip6+ with this fix?